src/DataTable/DataTable.svelte

```diff
@@ -200,7 +200,7 @@
     rows[row.id] = headerKeys.map((key, index) => ({
       key,
       value: resolvePath(row, key),
-      display: headers[index].display,
+      display: row.display ? row.display : headers[index].display,
```

Comment:
I would prefer the original behavior, where the `display` is optionally controlled by the header (and this PR now passes the original `row` to `display`).

Reply:
Ok, I just realized something: the `display` function isn't actually used in the header part.

Using it for headers isn't really in the scope of this PR, I'll rollback this specific change and keep the original behavior 👌